starter-code/scripts/article.js

```diff
@@ -19,10 +19,13 @@ Article.prototype.toHtml = function() {
   //   or say "(draft)" if it has no publication date:
   this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
   this.publishStatus = this.publishedOn ? 'published ' + this.daysAgo + ' days ago' : '(draft)';
-  // TODO: Use handlebars to render your articles!
+  // TODO: DONE Use handlebars to render your articles!
   //       - Select your template from the DOM.
   //       - Now "compile" your template with Handlebars.
   //       - Don't forget to return your template for this article.
+  var htmlTemp = $('#template').html();
+  var temp = Handlebars.compile(htmlTemp);
+  return temp(this);
```

Comment:
Great work grabbing a reference to your template and getting the HTML inside, compiling that template with Handlebars, and plugging in the data from your article instance into the template.